entrypoint.sh

```diff
@@ -1,12 +1,26 @@
 #!/bin/bash
 
-set -e
```

Comment:
**question:** @elainefigma , do we need to remove `set -e`? If I understand correctly, `-e` means " Exit immediately if a command exits with a non-zero status.", which seems like the correct behavior for this script.

Reply:
Ah, this is my bad. I put this back.